.github/workflows/tests.yml

```diff
@@ -328,4 +364,86 @@ jobs:
                 repo: variables.name,
                 body: issue_body,
                 title: title,
-            })
\ No newline at end of file
+            })
+
+  process-results:
+    needs: [runtime, benchmarks, stability]
+    name: Combine separate benchmark results
+    if: always() # && github.ref == 'refs/heads/main' && github.repository == 'coiled/coiled-runtime'
+    runs-on: ubuntu-latest
+    concurrency:
+      # Fairly strict concurrency rule to avoid stepping on benchmark db.
+      # Could eventually replace with a real db in coiled, RDS, or litestream
+      group: process-benchmarks
+      cancel-in-progress: false
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v4
+
+      - name: Install dependencies
+        run: pip install alembic
+
+      - uses: actions/download-artifact@v3
+        with:
+          path: benchmarks
+
+      - name: Download benchmark db
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.RUNTIME_CI_BOT_AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.RUNTIME_CI_BOT_AWS_SECRET_ACCESS_KEY }}
+          AWS_DEFAULT_REGION: us-east-2  # this is needed for boto for some reason
+          DB_NAME: benchmark.db
+        run: |
+          aws s3 cp s3://coiled-runtime-ci/benchmarks/$DB_NAME . || true
+
+      - name: Combine benchmarks
+        run: |
+          ls -lhR benchmarks
+          bash ci/scripts/combine-dbs.sh
+
+      - name: Upload benchmark db
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.RUNTIME_CI_BOT_AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.RUNTIME_CI_BOT_AWS_SECRET_ACCESS_KEY }}
+          AWS_DEFAULT_REGION: us-east-2  # this is needed for boto for some reason
+          DB_NAME: benchmark.db
+        run: |
+          aws s3 cp $DB_NAME s3://coiled-runtime-ci/benchmarks/
+
+      - name: Upload benchmark results as artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: benchmark.db
+          path: benchmark.db
+
+  static-site:
+    needs: process-results
+    name: Build static dashboards
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: benchmark.db
+
+      - name: Set up environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniforge-variant: Mambaforge
+          use-mamba: true
+          python-version: "3.9"
+          environment-file: ci/environment-dashboard.yml
+
+      - name: Generate dashboards
+        run: |
+          python dashboard.py
+
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@4.1.7
+        with:
+          branch: gh-pages
+          folder: static
```

Comment:
Can we put links to these pages somewhere? For example, a badge in the README

Reply:
Yep! I was avoiding doing that until we settled on how the pages are grouped/categorized. But I think now is a good time to do it.

Reply:
This is going to be a lot of badges, any objection to that?

Reply:
How difficult would it be to have a single landing page with links to all the individual python / runtime version / ... combination pages?

Reply:
Not too hard. Here is a look at all the combos as badges:

![image](https://user-images.githubusercontent.com/5728311/180282716-4677bc88-09c7-428c-9b66-7eca02e34243.png)